DigitalLearningSolutions.Web/Helpers/FilterableTagHelper.cs

```diff
@@ -130,6 +130,19 @@ public static IEnumerable<SearchableTagViewModel> GetCurrentTagsForCourseDelegat
                 tags.Add(new SearchableTagViewModel(CourseDelegateProgressRemovedFilterOptions.NotRemoved, true));
             }
 
+            if (courseDelegate.Completed.HasValue)
+            {
+                tags.Add(new SearchableTagViewModel(CourseDelegateCompletionFilterOptions.Complete));
+            }
+            else if (!courseDelegate.Completed.HasValue && !courseDelegate.Removed)
+            {
+                tags.Add(new SearchableTagViewModel(CourseDelegateCompletionFilterOptions.Incomplete));
+            }
+            else if (!courseDelegate.Completed.HasValue && courseDelegate.Removed)
+            {
+                tags.Add(new SearchableTagViewModel(CourseDelegateCompletionFilterOptions.Removed));
+            }

```

Comment:
I discussed with @SteveJacksonSoft and we thought Completed should take precedence, but I'm not wedded to the decision. I agree the overlap with Removed is worth scrutinising, I don't see the purpose of it here.

Reply:
Sorry, missed that we have an `is removed` filter.
I'm keen to make `is completed` an extra bool filter, but I've left [a comment for Kevin](https://softwire.slack.com/archives/C016Q1GR2G5/p1649426114277979?thread_ts=1647959140.959269&cid=C016Q1GR2G5) to check.

Reply:
He's agreed. Can we 
* Remove the "removed" status from the filter
* Add a `Completed` tag
* Rename the filter key according to our boolean filter conventions
* Modify the DelegateCourses logic so that Completions covers removed completed progress records (this might not be done yet - see the slack thread mentioned in the previous comment for more info; I'll update this comment when my calls are done and I get a chance to check if this is applicable yet).